examples/run/sycl/seeding_example_sycl.sycl

```diff
@@ -0,0 +1,261 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// SYCL include(s)
+#include <CL/sycl.hpp>
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+
+// Vecmem include(s)
+#include <vecmem/memory/host_memory_resource.hpp>
+#include <vecmem/memory/sycl/shared_memory_resource.hpp>
+
+// algorithms
+#include "../../../device/sycl/include/sycl/seeding/track_params_estimation.hpp"
+#include "../../algorithms/sycl/include/sycl/track_finding/seeding_algorithm.hpp"
+#include "seeding/track_params_estimation.hpp"
+#include "track_finding/seeding_algorithm.hpp"
+
+// io
+#include "io/csv.hpp"
+#include "io/reader.hpp"
+#include "io/writer.hpp"
+
+// Selector to pass to sycl::queue object for running on NVIDIA device
+class CUDADeviceSelector : public ::sycl::device_selector {
+    public:
+    int operator()(const ::sycl::device& device) const override {
+        return device.get_platform().get_backend() == ::sycl::backend::cuda
+                   ? 1
+                   : -1;
+    }
+};
```

Comment:
Please remove this custom selector from the code. There's no need to use any custom selector here. We can influence what device the application should run on, using the `SYCL_DEVICE_FILTER` environment variable.

Reply:
Done.